receiver/kubeletstatsreceiver/README.md

```diff
@@ -0,0 +1,65 @@
+# Kubelet Stats Receiver
+
+### Overview
+
+The Kubelet Stats Receiver pulls pod metrics from the API server on a kubelet
+and sends it down the metric pipeline for further processing.
+
+Status: beta
+
+### Configuration
+
+A kubelet runs on a kubernetes node and has an API server to which this
+receiver connects. To configure this receiver, you have to tell it how
+to connect and authenticate to the API server and how often to collect data
+and send it to the next consumer.
+
+There are two ways to authenticate, driven by the `auth_type` field: "tls" and
+"serviceAccount".
+
+TLS tells this receiver to use TLS for auth and requires that the fields
+`ca_cert_path`, `client_key_path`, and `client_cert_path` also be set.
+
+ServiceAccount tells this receiver to use the default service account token
+to authenticate to the kubelet API.
+
+```yaml
+receivers:
+  kubeletstats:
+    collection_interval: 20s
+    auth_type: "tls"
+    ca_cert_path: "/path/to/ca.crt"
+    client_key_path: "/path/to/apiserver.key"
+    client_cert_path: "/path/to/apiserver.crt"
```

Comment:
There is an upcoming change that uses different key names for client configs: https://github.com/open-telemetry/opentelemetry-collector/pull/988/files/405db7ad13c8feeaeb65142081b04b48e407eaf6#diff-b99ea1d4f9c67b0e6a0dc0199de54d66
It may be useful for you to sync with @ccaraman on this. For now you can probably just use the same field names to avoid breaking changes later once @ccaraman change is merged.

Reply:
Using the same field names for now. Thanks.